### ISSUE TITLE
How to produce spectrogram segmentation maps?

### ISSUE THREAD

**Issue by bartmch** [uvaydovd/spectrum_sensing_stitching#4]
I read your article and implementation with great interest. After inference, I get logits (model output) for each class (5) at each timestep (1024). I don't understand and I'm not able to find the implementation of how you created the segmented spectrogram in both time and frequency, eg fig 9.
In other words, how can I re-create the semantic segmentation maps in both time and frequency on the spectrogram?
